contrib/docker/scripts/build.sh

```diff
@@ -16,7 +16,14 @@ do
     [ -f $p ] && source $p
 done
 
-git clone https://github.com/puppetlabs/libral
+if [ ! -d libral ]; then
+    echo "Cloning libral"
+    git clone https://github.com/puppetlabs/libral
+else
+    echo "Building from existing libral directory"
```

Comment:
I'd left the mounted repo unaltered, with the assumption that the user would want to build using only what they currently had in their workspace rather than have us alter the workspace (beyond creating a build dir).